src/test/java/com/fryrank/TestConstants.java

```diff
@@ -28,6 +28,8 @@ public class TestConstants {
 
     public static final String TEST_USERNAME = "testflush";
 
+    public static final String ISO_DATE_TIME = "isoDateTime";
```

Comment:
Reuse the constant you created for the implementation code instead of creating an identical one for the tests.